### PR TITLE
Upgrade kubernetes-client

### PR DESCRIPTION
**Pull request by luissajunior**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix

* **What is the current behavior?** (You can also link to an open issue here)
dojot/dojot#1548

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

**Comment by mprevide** [comment]
Try using forks in the next PRs

**Comment by mprevide** [comment]
Don't forget about 0.5.0 after merge this.

File: orchestrator/nodes/kubernetesRemoteNode/index.js [hunk 1]
```diff
@@ -145,20 +151,22 @@ class DataHandler extends RemoteNode {
...
         return;
       }
       logger.debug(`Retrieving current deployment...`, { filename: 'kb8sRemoveNode' });
       logger.debug(`Sending request to server...`, { filename: 'kb8sRemoveNode' });
-      this.ext.namespaces("dojot").deployments("").get().then((value) => {
+      this.api.apis.apps.v1.namespaces("dojot").deployments('').get().then((value) => {
         let tempDeploymentNames = [];
-        for (let deployment of value.items) {
+
+        //for(let i = 0; i < value.items.length; i++){
```
Comment (mprevide) [inline orchestrator/nodes/kubernetesRemoteNode/index.js:163]: Remove commented code

File: orchestrator/nodes/kubernetesRemoteNode/index.js [hunk 2]
```diff
@@ -145,20 +151,22 @@ class DataHandler extends RemoteNode {
...
-        for (let deployment of value.items) {
+
+        //for(let i = 0; i < value.items.length; i++){ 
+        for (let deployment of value.body.items) {
           tempDeploymentNames.push(deployment.metadata.name);
         }
         // Get only those ones created by flowbroker
         this.deploymentNames = tempDeploymentNames.filter((name) => (name.match(/^flownode-.*/) != null));
-        logger.debug(`Current flowbroker deployments are: ${util.inspect(value, { depth: null })}`, { filename: 'kb8sRemoveNode' });
+        //logger.debug(`Current flowbroker deployments are: ${util.inspect(value, { depth: null })}`, { filename: 'kb8sRemoveNode' });
```
Comment (mprevide) [inline orchestrator/nodes/kubernetesRemoteNode/index.js:169]: Uncomment

File: orchestrator/nodes/kubernetesRemoteNode/index.js [hunk 3]
```diff
@@ -111,7 +117,7 @@ class DataHandler extends RemoteNode {
       this.token = "";
       this.host = "";
       this.api = null;
-      this.ext = null;
+      //this.ext = null;
```
Comment (mprevide) [inline orchestrator/nodes/kubernetesRemoteNode/index.js:120]: Remove commented code

File: orchestrator/nodes/kubernetesRemoteNode/index.js [hunk 4]
```diff
@@ -93,12 +99,12 @@ class DataHandler extends RemoteNode {
...
       options.version = "v1";
-      this.api = new k8s.Core(options);
-      options.version = "v1beta1";
-      this.ext = new k8s.Extensions(options);
-
-      logger.debug(`Using kubernetes API server @ ${this.host}`, { filename: 'kb8sRemoveNode' });
+      this.api = new k8s.Client({ version: '1.13' });
+      //this.api = new k8s.Core(options);
+      options.version = "v1";
+      //this.ext = new k8s.Extensions(options);
```
Comment (mprevide) [inline orchestrator/nodes/kubernetesRemoteNode/index.js:105]: Remove commented code

File: orchestrator/nodes/kubernetesRemoteNode/index.js [hunk 5]
```diff
@@ -93,12 +99,12 @@ class DataHandler extends RemoteNode {
       // Initialize API access
       let options = this.getDefaultGroupOptions();
       options.version = "v1";
-      this.api = new k8s.Core(options);
-      options.version = "v1beta1";
-      this.ext = new k8s.Extensions(options);
-
-      logger.debug(`Using kubernetes API server @ ${this.host}`, { filename: 'kb8sRemoveNode' });
+      this.api = new k8s.Client({ version: '1.13' });
+      //this.api = new k8s.Core(options);
```
Comment (mprevide) [inline orchestrator/nodes/kubernetesRemoteNode/index.js:103]: Remove commented code

**Comment by mprevide** [comment]
I think @andersonluisribeiro can review these changes better than me.

**Review by andersonluisribeiro — COMMENTED** [review]
Was this tested using ansible? I think k8s role must be changed for this to work.

**Comment by luissajunior** [comment]
> Was this tested using ansible? I think k8s role must be changed for this to work.

That's true. I change locally for testing. Im gona make the commit in ansible. Tks

**Comment by luissajunior** [comment]
> > Was this tested using ansible? I think k8s role must be changed for this to work.
> 
> That's true. I change locally for testing. Im gona make the commit in ansible. Tks

PR created for ansible
dojot/ansible-dojot#67

**Review by andersonluisribeiro — APPROVED** [review]
:+1:

**Review by mprevide — APPROVED** [review]
Don't forget about 0.5.0 after merge this.

**Pull request merged**